Open_Science_New.html

```diff
@@ -376,56 +428,44 @@ <h2>Privacy Concerns</h2>
 		  </td>
 	      </tr></table>
 	    </section>
-	  </section>
-	  
-	  <!-- Brainhack -->
+
 	  <section data-transition="fade">
 	    <table class="reveal">
-	      <h2><a href="">Hackathons</a></h2>
 	      <tr>
 		<td width="75%" style="vertical-align: middle">
-		  <img src="images/C-BASE.png">
+		  <a href="http://loris.ca/LORIS_Governance_1904.pdf"><img src="images/LORISgov.png"></a>
 		</td>
 		<td width="25%" style="vertical-align: middle">
-		  <p> Important value of hackathons to data sharing initiatives</p>
-		  <img src="images/Brainhack.png">
+		  <p>See how MCIN adheres to Open Science ethics.<br>
+              <br>More on LORIS next...</p>
 		</td>
 	    </tr></table>
 	  </section>
-	  
-	  <!-- Our implementation -->
-	  <section>
-	    <section data-background="images/mni_eco_info_5.jpg" data-background-size="100%">
-	    </section>
-	    
-	    <!-- MNI PAPER -->	
-	    <section data-transition="fade">
-              <a href="https://www.researchgate.net/publication/281780543_The_MNI_data-sharing_and_processing_ecosystem" target="_blank"><img src="images/MNI_ecosystem_paper.png"></a>
-	    </section>
-	    
-	    <!-- WHAT IS LORIS? -->
+	  </section>
+
+        <!-- WHAT IS LORIS? -->
 	    <section>
-	      <a href="http://loris.ca/" target="_blank"><h2>WHAT IS LORIS?</h2></a>
+            <section>
+	      <a href="http://loris.ca/" target="_blank"><h2>Introducing... LORIS!</h2></a>
 	      <table class="reveal">
 		<tr>
 		  <td width="50%" style="vertical-align: middle">
-		    <a href="http://loris.ca/" target="_blank"><img src="images/loris-new-white.png"></a>	   
+		    <a href="http://loris.ca/" target="_blank"><img src="images/loris-new-white.png"></a>
 		    <p><small> “LORIS is a modular and <font color="f27c30">extensible</font> web-based data management system that integrates all aspects of a <font color="f27c30">multi-center</font> study: from <font color="f27c30">heterogeneous data acquisition</font> (imaging, clinical, behavior, genetics) to storage, <font color="f27c30">processing</font> and ultimately <font color="f27c30">dissemination</font>.”</small></p>
```

Comment:
make/add 2 intro slides in this column (starting with/replacing this slide) to reflect the content in [these "LORIS in a nutshell" slides](https://docs.google.com/presentation/d/1_KXSFXwXeX7HYv2thv7E3hc7CBxyGAiWMBxzjuN0ZZA/edit#slide=id.g6185f6c069_3_78)
(don't edit those google slides, but copying from them is fine)